data/json/items/armor/bespoke_armor/custom_gloves.json

```
@@ -330,7 +330,7 @@
     "id": "gloves_h20survivor_thick",
     "type": "ARMOR",
     "category": "armor",
-    "name": { "str": "pair of Kevlar wetsuit gloves", "str_pl": "pairs of Kevlar wetsuit gloves" },
+    "name": { "str": "thick pair of Kevlar wetsuit gloves", "str_pl": "thick pairs of Kevlar wetsuit gloves" },
```

```suggestion
    "name": { "str": "pair of thick Kevlar wetsuit gloves", "str_pl": "pairs of thick Kevlar wetsuit gloves" },
```

The way you renamed them does not, in fact, match the thick Kevlar naming convention